src/components/EditorDrawer.jsx

```diff
@@ -3,7 +3,7 @@ import { Drawer as VaulDrawer } from 'vaul';
 import { useDispatch, useSelect } from '@wordpress/data';
 
 export function EditorDrawer( { children } ) {
-	const buttonLabel = 'GraphQL IDE';
+	const buttonLabel = '🚀';
```

Comment:
I'm cool with this as a short-term solution, but I anticipate re-visiting this at some point as I don't think the 🚀 _really_ conveys to anyone other than you an I what this button really is meant for 🤔 

But, at the moment I don't have another suggestion. Just thinking we _might_ revisit this at some point. 🤷🏻‍♂️